<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Abhijith" failures="1" tests="2" name="Test" time="15.728" errors="0" timestamp="2025-05-31T15:44:27 IST">
  <testcase classname="computers.TC_DWS_010_Test" name="clickOnComputers" time="0.867"/>
  <testcase classname="books.TC_DWS_001_Test" name="clickOnBooks" time="1.014">
    <failure type="java.lang.AssertionError" message="expected [https://demowebshop.tricentis.] but found [https://demowebshop.tricentis.com/books]">
      <![CDATA[java.lang.AssertionError: expected [https://demowebshop.tricentis.] but found [https://demowebshop.tricentis.com/books]
at books.TC_DWS_001_Test.clickOnBooks(TC_DWS_001_Test.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- clickOnBooks -->
  <testcase classname="genericutility.BaseClass" name="@BeforeSuite reportConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite reportConfig -->
  <testcase classname="genericutility.BaseClass" name="@AfterSuite reportBackup" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite reportBackup -->
</testsuite> <!-- Test -->
